docker/Dockerfile

```diff
@@ -135,19 +134,24 @@ RUN --mount=type=bind,from=rosdep-depend,source=/autoware/src,target=/autoware/s
     --mixin release compile-commands ccache \
   && du -sh ${CCACHE_DIR} && ccache -s
 
-CMD ["/bin/bash"]
-
-FROM autoware-universe as devel
+FROM autoware-core as devel
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Install development tools and artifacts
+# Install rosdep dependencies for universe packages
+COPY --from=rosdep-depend /rosdep-universe-depend-packages.txt /tmp/rosdep-universe-depend-packages.txt
+# hadolint ignore=SC2002
 RUN --mount=type=ssh \
-  --mount=type=cache,target=/var/cache/apt,sharing=locked \
-  ./setup-dev-env.sh -y --module dev-tools openadkit \
-  && pip uninstall -y ansible ansible-core \
-  && apt-get autoremove -y && rm -rf "$HOME"/.cache
+  apt-get update \
+  && cat /tmp/rosdep-universe-depend-packages.txt | xargs apt-get install -y --no-install-recommends \
+  && apt-get autoremove -y && apt-get clean -y && rm -rf /var/lib/apt/lists/* "$HOME"/.cache
```

Comment:
You need to install `rosdep-universe-depend-packages.txt` packages after this PR is merged. I think it's not efficient and simple.